src/jit/importer.cpp

```diff
@@ -13801,13 +13821,15 @@ bool Compiler::impReturnInstruction(BasicBlock *block, int prefixFlags, OPCODE &
                 {
                     // Same as !IsHfa but just don't bother with impAssignStructPtr.
 #else // !defined(_TARGET_ARM_)
-                SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-                eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &structDesc);
-                if (structDesc.passedInRegisters)
+                ReturnTypeDesc retTypeDesc;
```

Comment:
There is no indentation problem.  The If condition is different for Arm and x64 Unix.  In case of x64 Unix there are 3 lines of code before the if condition.